### ISSUE TITLE
optimal model is not saved

### ISSUE THREAD

**Issue by dagap** [eriklindernoren/PyTorch-YOLOv3#212]
In your training code, you actually never save the optimal model, just the checkpoints. I see that the output folder is created but never populated. I guess one just needs to keep track of the lowest loss and save that model in the end.
